pimd/pim_ssm.c

```diff
@@ -72,8 +72,14 @@ void pim_ssm_prefix_list_update(struct pim_instance *pim,
 static int pim_is_grp_standard_ssm(struct prefix *group)
 {
 	pim_addr addr = pim_addr_from_prefix(group);
-
+#if PIM_IPV == 4
```

Comment:
This if else is not required since pim_addr_ssm function is already defined based on v4 and v6 flag

Reply:
Sure, thank you.